bundle/regal/rules/idiomatic/no_defined_entrypoint_test.rego

```diff
@@ -3,7 +3,6 @@ package regal.rules.idiomatic["no-defined-entrypoint_test"]
 import future.keywords.if
 import future.keywords.in
 
-import data.regal.ast
```

Comment:
I wonder if we could have a rule for unused imports?

Reply:
We could, but it's in OPA strict mode already, which is why I've chosen not to duplicate this here. Running `opa check --strict` on Regal is a bit clunky though, as we need to provide a capabilities file including our custom built-in functions. We currently don't produce such a file, but we probably should. 

Another option would be to embed `opa check --strict` as a linter rule similar to the `opa fmt` one, but I'm [not pleased](https://github.com/open-policy-agent/opa/issues/5815) with the compiler API.

Seeing as this is likely going to be the default for OPA in some future release, the status quo remains :)

Reply:
👍 I'd forgotten about that. Thanks!

Reply:
Created https://github.com/StyraInc/regal/issues/369 :)